test/gtest/ucp/test_ucp_fence.cc

```diff
@@ -212,3 +229,20 @@ UCS_TEST_P(test_ucp_fence64, atomic_add_fadd) {
 }
 
 UCP_INSTANTIATE_TEST_CASE(test_ucp_fence64)
+
+class test_ucp_fence64_mt : public test_ucp_fence {
+public:
+    static ucp_params_t get_ctx_params() {
+        ucp_params_t params = test_ucp_fence::get_ctx_params();
+        params.features |= UCP_FEATURE_AMO64;
+        params.thread_mode = UCS_THREAD_MODE_MULTI;
+        return params;
+    }
+};
+
+UCS_TEST_P(test_ucp_fence64_mt, atomic_add_fadd) {
```

Comment:
this test is already run with multiple threads, here I run it with THREAD_MULTI mode